core/src/poh_service.rs

```diff
@@ -122,6 +122,10 @@ impl PohService {
                         ticks_per_slot,
                         hashes_per_batch,
                         record_receiver,
+                        Self::target_ns_per_tick(
```

Comment:
unrelated to this PR, but it seems like since `target_ns_per_tick()` is just subtracting a constant number from `poh_config.target_tick_duration` every time, might make sense to just move this into the initialization of `poh_config.target_tick_duration`